skill-package/interactionModels/custom/de-DE.json

```diff
@@ -29,7 +29,10 @@
                 ],
                 "samples": [
                   "{item}",
+                  "mach {item} drauf",
```

Comment:
This does not sound like proper German to me.

"setz {item} drauf" would be more appropriate.

Similar below

Reply:
We'll here "im schwäbischen" we talk like this. I don't think there's anything wrong with adding samples like this. Alexa normally also understands this.
But if you only want "Hochdeutsch" in there, feel free to make changes or reject the PR.